### PR TITLE
docs: add simulate-aws-chaos

### PR DESCRIPTION
**Pull request by TomShawn**
Signed-off-by: TomShawn <41534398+TomShawn@users.noreply.github.com>

This PR is translated from https://github.com/chaos-mesh/website-zh/pull/30.

### CODE REVIEW

File: docs/simulate-aws-chaos.md [hunk 1]
```diff
@@ -1,6 +1,167 @@
...
-## TODO
+This document describes how to use Chaos Mesh to simulate AWS faults.
+
+## AwsChaos introduction
+
+AwsChaos can help you simulate fault scenarios on the specified AWS instance. Currently, AwsChaos supports the following fault types:
+
+- **Ec2 Stop**: Stops the specified EC2 instance.
+- **Ec2 Restart**: Restarts the specified EC2 instance.
+- **Detach Volume**: Uninstalls the storage volume from the specified EC2 instance.
```
Comment (Liuxiaozhen12) [inline docs/simulate-aws-chaos.md:14]:
```suggestion
- **Ec2 Stop**: stops the specified EC2 instance.
- **Ec2 Restart**: restarts the specified EC2 instance.
- **Detach Volume**: uninstalls the storage volume from the specified EC2 instance.
```

File: docs/simulate-aws-chaos.md [hunk 2]
```diff
@@ -1,6 +1,167 @@
...
+
+   ```bash
+    kubectl port-forward -n chaos-testing svc/chaos-dashboard 2333:2333
+   ```
+
+   You can then access the dashboard via [`http://localhost:2333`](http://localhost:2333) in your browser.
+
+:::
+
+1. Open Chaos Dashboard, and click the "**NEW EXPERIMENT**" button on the page to create a new experiment:
```
Comment (Liuxiaozhen12) [inline docs/simulate-aws-chaos.md:56]:
```suggestion
1. Open Chaos Dashboard, and click **NEW EXPERIMENT** on the page to create a new experiment:
```

File: docs/simulate-aws-chaos.md [hunk 3]
```diff
@@ -1,6 +1,167 @@
...
+
+    ![img](./img/create-new-exp.png)
+
+2. In the "**Choose a Target**" area, choose **AWS FAULT** and select a specific behavior, such as **STOP EC2**.
+
+3. Fill out the experiment information, and specify the experiment scope and the scheduled experiment duration:
```
Comment (Liuxiaozhen12) [inline docs/simulate-aws-chaos.md:62]:
```suggestion
3. Fill out the experiment information, and specify the experiment scope and the scheduled experiment duration.
```

File: docs/simulate-aws-chaos.md [hunk 4]
```diff
@@ -1,6 +1,167 @@
...
+
+    ```bash
+    kubectl apply -f awschaos-detach-volume.yaml
+    ```
+
+### Field description
+
+The following table shows the fields in the YAML configuration file.
+
+| Parameter   | Type   | Note                                                                                                                                                                                                                                                                                                                                                                        | Default value | Required | Example              |
```
Comment (Liuxiaozhen12) [inline docs/simulate-aws-chaos.md:157]:
```suggestion
| Parameter   | Type   | Description                                                                                                                                                                                                                                                                                                                                                                        | Default value | Required | Example              |
```

**Review by Liuxiaozhen12 — COMMENTED** [review]
rest lgtm

**Comment by TomShawn** [comment]
@fewdan PTAL

File: docs/simulate-aws-chaos.md [hunk 5]
```diff
@@ -1,6 +1,167 @@
...
+title: Simulate AWS Faults
+sidebar_label: Simulate AWS Faults
 ---
 
-## TODO
+This document describes how to use Chaos Mesh to simulate AWS faults.
+
+## AwsChaos introduction
+
+AwsChaos can help you simulate fault scenarios on the specified AWS instance. Currently, AwsChaos supports the following fault types:
```
Comment (cwen0) [inline docs/simulate-aws-chaos.md:10]: s/AwsChaos/AWSChaos/g, I will update it on Chinese document.

Comment (cwen0) [reply docs/simulate-aws-chaos.md:10]: https://github.com/chaos-mesh/website-zh/pull/60

File: docs/simulate-aws-chaos.md [hunk 6]
```diff
@@ -1,6 +1,167 @@
...
-title: 模拟 AWS 故障
-sidebar_label: 模拟 AWS 故障
+title: Simulate AWS Faults
+sidebar_label: Simulate AWS Faults
 ---
 
-## TODO
+This document describes how to use Chaos Mesh to simulate AWS faults.
+
+## AwsChaos introduction
```
Comment (TomShawn) [inline docs/simulate-aws-chaos.md:8]:
```suggestion
## AWSChaos introduction
```

File: docs/simulate-aws-chaos.md (hunk 5 above)
Comment (TomShawn) [reply docs/simulate-aws-chaos.md:10]:
```suggestion
AWSChaos can help you simulate fault scenarios on the specified AWS instance. Currently, AWSChaos supports the following fault types:
```

File: docs/simulate-aws-chaos.md [hunk 7]
```diff
@@ -1,6 +1,167 @@
...
+
+## Create experiments using the YAML file
+
+### An `ec2-stop` configuration example
+
+1. Write the experiment configuration to the `awschaos-ec2-stop.yaml` file, as shown below:
+
+    ```yaml
+    apiVersion: chaos-mesh.org/v1alpha1
+    kind: AwsChaos
```
Comment (TomShawn) [inline docs/simulate-aws-chaos.md:74]:
```suggestion
    kind: AWSChaos
```

File: docs/simulate-aws-chaos.md [hunk 8]
```diff
@@ -1,6 +1,167 @@
...
+    kubectl apply -f awschaos-ec2-stop.yaml
+    ```
+
+### An `ec2-start` configuration example
+
+1. Write the experiment configuration to the `awchaos-ec2-restot.yaml` file:
+
+    ```yaml
+    apiVersion: chaos-mesh.org/v1alpha1
+    kind: AwsChaos
```
Comment (TomShawn) [inline docs/simulate-aws-chaos.md:102]:
```suggestion
    kind: AWSChaos
```

File: docs/simulate-aws-chaos.md [hunk 9]
```diff
@@ -1,6 +1,167 @@
...
+    kubectl apply -f awschaos-ec2-restart.yaml
+    ```
+
+### A `detail-volume` configuration example
+
+1. Write the experiment configuration to the `awschaos-detach-volume.yaml` file:
+
+    ```yaml
+    apiVersion: chaos-mesh.org/v1alpha1
+    kind: AwsChaos
```
Comment (TomShawn) [inline docs/simulate-aws-chaos.md:129]:
```suggestion
    kind: AWSChaos
```

**Comment by TomShawn** [comment]
@cwen0 PTAL again, thanks!

File: docs/simulate-aws-chaos.md [hunk 10]
```diff
@@ -1,6 +1,167 @@
...
 ---
 
-## TODO
+This document describes how to use Chaos Mesh to simulate AWS faults.
+
+## AWSChaos introduction
+
+AWSChaos can help you simulate fault scenarios on the specified AWS instance. Currently, AWSChaos supports the following fault types:
+
+- **Ec2 Stop**: stops the specified EC2 instance.
```
Comment (cwen0) [inline docs/simulate-aws-chaos.md:12]: s/Ec2/EC2/g

File: docs/simulate-aws-chaos.md [hunk 11]
```diff
@@ -1,6 +1,167 @@
...
+- **Ec2 Restart**: restarts the specified EC2 instance.
```
Comment (cwen0) [inline docs/simulate-aws-chaos.md:13]: ditto

**Comment by cwen0** [comment]
> @cwen0 PTAL again, thanks!

@TomShawn PTAL

**Review by Liuxiaozhen12 — APPROVED** [review]

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**